index.html

```diff
@@ -185,5 +185,38 @@ <h1>Skills</h1>
                  </ul>
              </div>
           </section>
+          <footer class="footer-items">
+             <div class="feedback">
+              <p class="feedback-text">
+                  I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+              </p>                    
+              <form action="https://formspree.io/your-email" method="POST">
+                  <input class="form-input" type="text" id="fullname" name="fullname" placeholder="Full name" maxlength="30" required>
+                  <input class="form-input" type="email" id="email" name="email" placeholder="Email address" required>
+                  <textarea id="message" name="message" rows="5" placeholder="Write me something..." maxlength="500" required></textarea>                        
+                  <input class="submit-btn" type="submit" value="Get in touch">
```

Comment:
- [ ] Great job on adding styles for the enabled state of the button! However, it's important to also include styles for the hover and pressed states to match the design guidelines. Please ensure that you add appropriate styles to enhance the interactivity and visual feedback of the button when hovered or pressed. This will provide a more polished and cohesive user experience. Keep up the good work! 👍🎉

![Uploading image.png…]()